### PR TITLE
Add jQuery Bootgrid

### PR DESCRIPTION
**Pull request by Lykegenes**

### CODE REVIEW

File: files/jquery.bootgrid/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "jQuery Bootgrid",
+  "repo": "rstaib/jquery-bootgrid",
+  "files": {
+    "basePath": "dist/",
+    "include": ["*.js"]
```
Comment (jimaek) [inline files/jquery.bootgrid/update.json]: Use `*.min.*` instead to include the css file as well

**Comment by Lykegenes** [comment]
Done! I thought you only served Javascript files.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by jimaek** [comment]
Thank you

**Pull request closed without merging**